pkg/registry/apis/secret/encryption/manager/manager.go

```diff
@@ -95,7 +97,11 @@ func NewEncryptionManager(
 
 func (s *EncryptionManager) InitProviders() (err error) {
 	s.pOnce.Do(func() {
-		s.providers, err = s.kmsProvidersService.Provide()
+		providers, _ := s.kmsProvidersService.Provide()
+		s.providers = make(map[encryption.ProviderID]encryption.Provider, len(providers))
```

Comment:
slightly annoying but I want to keep the domains as separate as possible so we don't disturb OSS Grafana or have extra annoying dependencies there